client/packages/system/src/Stock/api/hooks/useStockLine.ts

```diff
@@ -186,6 +187,7 @@ const useUpdate = (id: string) => {
         onHold,
         sellPricePerPack,
         location: setNullableInput('id', location),
+        itemVariantId: setNullableInput('id', itemVariant),
```

Comment:
i thought was removing ability to edit item variant?

Reply:
Opps, thought I added to body above. No, Mark said keep edits available but disable edits on variants themselves if they have been assigned..

Reply:
ohh so only if set gotcha 🙏